examples/plotting/file/grid.py

```diff
@@ -1,3 +1,10 @@
+'''  A grid plot that shows four figures that uses different glyphs.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.layouts.gridplot, bokeh.plotting.Figure.line, bokeh.plotting.Figure.annular_wedge, bokeh.plotting.Figure.bezier, bokeh.plotting.Figure.quad,
```

Comment:
extra trailing comma
```suggestion
    :apis: bokeh.layouts.gridplot, bokeh.plotting.Figure.line, bokeh.plotting.Figure.annular_wedge, bokeh.plotting.Figure.bezier, bokeh.plotting.Figure.quad
```